tests/unit/conditions/test_core_shims.py

```diff
@@ -24,4 +24,4 @@ def test_conditional_message_kit_serialization(enacted_federated_policy, ERC1155
     )
     mk_bytes = bytes(mk)
     restored_mk = MessageKit.from_bytes(mk_bytes)
-    assert bytes(restored_mk.lingos) == bytes(ERC1155_balance_condition)
+    assert bytes(restored_mk.conditionss) == bytes(ERC1155_balance_condition)
```

Comment:
```suggestion
    assert bytes(restored_mk.conditions) == bytes(ERC1155_balance_condition)
```

Reply:
Yep - see <https://github.com/nucypher/nucypher/pull/2961#discussion_r957407413> in #2961 . I don't want to touch that code and cause conflicts.